airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/default_paginator.py

```diff
@@ -160,3 +160,68 @@ def _get_request_options(self, option_type: RequestOptionType) -> Mapping[str, A
             if option_type != RequestOptionType.path:
                 options[self.page_size_option.field_name] = self.pagination_strategy.get_page_size()
         return options
+
+
+class PaginatorTestReadDecorator(Paginator):
+    """
+    In some cases, we want to limit the number of requests that are made to the backend source. This class allows for limiting the number of
+    pages that are queried throughout a read command.
+    """
+
+    _PAGE_COUNT_BEFORE_FIRST_NEXT_CALL = 1
+    _DEFAULT_PAGINATION_LIMIT = 5
+
+    def __init__(self, decorated, maximum_number_of_pages: int = None):
+        self._decorated = decorated
+        self._page_count = self._PAGE_COUNT_BEFORE_FIRST_NEXT_CALL
+        self._maximum_number_of_pages = maximum_number_of_pages if maximum_number_of_pages else self._DEFAULT_PAGINATION_LIMIT
```

Comment:
Instead of `maximum_number_of_pages: int = None` and the ternary, can we just set the default for `maximum_number_of_pages: int = _DEFAULT_PAGINATION_LIMIT` directly in the signature?